keras_nlp/models/bert.py

```diff
@@ -319,6 +377,26 @@ def BertBase(name=None, trainable=True):
         trainable=trainable,
     )
 
-    # TODO(jbischof): add some documentation or magic to load our checkpoints
-    # TODO(jbischof): attach the tokenizer
+    # TODO(jbischof): consider changing format from `h5` to
+    # `tf.train.Checkpoint` once
+    # https://github.com/keras-team/keras/issues/16946 is resolved
+    if weights:
+        filepath = keras.utils.get_file(
+            weights,
```

Comment:
It looks like this will download to a file with a path `~/.keras/models/bert_base_uncased`, could we download instead to something like `~/.keras/models/bert_base_uncased/model.h5` using `cache_subdir`?

I think it would be nice for filepath to have an extension indicating the type.

Reply:
Is this what you're asking for?
```python
 filepath = keras.utils.get_file(
            model.h5,
            BASE_PATH + weights + "/model.h5",
            cache_subdir="models/bert_base/",
            file_hash=checkpoints["bert_base"][weights]["md5"],
        )
```

Reply:
sgtm!

Reply:
Checked the result and verified file structure:
```
bischof@bischof:~$ ls -l .keras/models/bert_base/*
.keras/models/bert_base/cased_en:
total 423320
-rw-r----- 1 bischof primarygroup 433474808 Aug 26 19:06 model.h5

.keras/models/bert_base/uncased_en:
total 427900
-rw-r----- 1 bischof primarygroup 438162680 Aug 26 19:06 model.h5
```